### PR TITLE
Docker dev

### PR DESCRIPTION
**Pull request by ssalinas**
Much faster dev/test environment using docker. Run docker-compose up and you have a working sinuglarity cluster. Want to add Baragon to this as well as a way to build and restart quicker for dev

### CODE REVIEW

**Comment by ssalinas** [comment]
@tpetr can you give this a go when you get a chance
`docker-compose pull` to update images `docker-compose up` to launch everything
with boot2docker, singularity ui should be available at http://192.168.59.103:7099/singularity

**Comment by ssalinas** [comment]
rebuilt the public image for this, the ui is working better now

**Comment by ssalinas** [comment]
@tpetr added some *try it out* documentation in our readme for this and got the compose configuration working so that it doesn't matter if you are on boot2docker or not. Should be good to merge unless you want any changes in the docs

File: docker/singularity/start.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+PATH=/usr/local/singularity/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/lib64/qt-3.3/bin:/usr/local/bin:/bin:/usr/bin
+MESOS_HOME=/usr/local
+MESOS_NATIVE_LIBRARY=/usr/local/lib/libmesos.so
+PORT=7092
```
Comment (tpetr) [inline docker/singularity/start.sh]: in the future, let's not hardcode this

**Comment by tpetr** [comment]
LGTM once extra line is removed

**Pull request merged**